public/js/whiteboard.js

```diff
@@ -22,12 +22,18 @@ function updateCrew(crewResponse) {
 }
 
 function updateNotes(noteResponse) {
+    let emoji = new EmojiConvertor();
+    emoji.init_env();
+    emoji.replace_mode = 'unified';
+    emoji.allow_native = true;
```

Comment:
Please make a helper function with the `EmojiConvertor` constructor call and the three settings for the `emoji` object as the same setup is used in multiple places.